client/next.js

```diff
@@ -10,18 +10,22 @@ const {
   __NEXT_DATA__: { component, errorComponent, props, ids, err }
 } = window
 
-const Component = evalScript(component).default
-const ErrorComponent = evalScript(errorComponent).default
+document.addEventListener('DOMContentLoaded', () => {
+  const Component = evalScript(component).default
+  const ErrorComponent = evalScript(errorComponent).default
 
-export const router = new Router(window.location.href, {
```

Comment:
Earlier, we've export the router from this module.
Currently we can't do it since we defer loading that.

I think what we don't need export in the future as we support it with `next/router` singleton.